src/importexport/musicxml/internal/musicxml/importmxmlpass1.cpp

```diff
@@ -480,7 +480,7 @@ track_idx_t MusicXMLParserPass1::trackForPart(const QString& id) const
 
 Fraction MusicXMLParserPass1::getMeasureStart(const int i) const
 {
-    if (0 <= i && i < m_measureStart.size()) {
+    if (0 <= i && i < static_cast<int>(m_measureStart.size())) {
```

Comment:
The parameter above should havwe been `const size_t i`, then no case is needed

Reply:
Yes, but I didn't want to go over all that mess. 😉 There is definitely more to clean up.